jupyter/minimal/ubi9-python-3.12/Dockerfile.cpu

```diff
@@ -1,7 +1,12 @@
+#########################
+# configuration args    #
+#########################
+ARG BASE_IMAGE
+
 ####################
-# base             #
+# cpu-base         #
 ####################
-FROM registry.access.redhat.com/ubi9/python-312:latest AS base
+FROM ${BASE_IMAGE} AS cpu-base
```

Comment:
I'm thinking if it would not make sense to define

* CPU_BASE_IMAGE
* CUDA_BASE_IMAGE
* ROCM_BASE_IMAGE

so that they can be all defined in a single .conf file alltogether

may be a bit easier to maintain later

but no strong preference either way

Reply:
Yes, I thought the same, and it makes sense. However, I moved to using a single .conf file for each image so that, if we decide to add specific parameters in the future, we’ll have more flexibility.